include/deal.II/base/vectorization.h

```diff
@@ -502,7 +500,7 @@ class VectorizedArray
   }
 
   /**
-   * Load @p n_array_elements from memory into the calling class, starting at
+   * Load @p size() from memory into the calling class, starting at
```

Comment:
The old text was broken, but it got worse now. Suggestion:
```suggestion
   * Load @p size() data items from memory into the calling class, starting at
```